final-project/index.html

```diff
@@ -1,14 +1,371 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
   <link rel="stylesheet" href="./dist/styles.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.2/css/all.css" integrity="sha384-/rXc/GQVaYpyDdyxK+ecHPVYJSN9bmVFBvjA/9eOB+pb3F2w2N6fc5qB9Ew5yIns"
+    crossorigin="anonymous">
+  <link href="https://fonts.googleapis.com/css?family=Quicksand|Roboto" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+  <script src="./scripts/blog.js"></script>
+  <script src="./scripts/faq.js"></script>
+  <script src="./scripts/sponsors.js"></script>
+
+  <script>
+    window.onload = function() {
+      populateBlog();
```

Comment:
All of these functions are in the global scope, a good practice is to avoid this by creating modules, importing and exporting them in js files. When done this way another js file can overwrite those functions having an unpredictable behavior.